src/event_loop.rs

```diff
@@ -224,14 +224,26 @@ impl<T> EventLoop<T> {
     ///   (that Rust doesn't see) that will also mean that the rest of the function is never executed
     ///   and any values not passed to this function will *not* be dropped.
     ///
-    ///   Web applications are recommended to use `spawn()` instead of `run()` to avoid the need
+    ///   Web applications are recommended to use
+    #[cfg_attr(
+        wasm_platform,
+        doc = " [`spawn()`][crate::platform::web::EventLoopExtWebSys::spawn()] "
+    )]
+    #[cfg_attr(not(wasm_platform), doc = " `spawn()`[^1] ")]
+    /// instead of [`run()`] to avoid the need
     ///   for the Javascript exception trick, and to make it clearer that the event loop runs
     ///   asynchronously (via the browser's own, internal, event loop) and doesn't block the
     ///   current thread of execution like it does on other platforms.
     ///
     ///   This function won't be available with `target_feature = "exception-handling"`.
     ///
-    /// [`set_control_flow()`]: EventLoopWindowTarget::set_control_flow
+    #[cfg_attr(
+        not(wasm_platform),
+        doc = "[^1]: `spawn()` is only available on `wasm` platforms."
+    )]
+    ///
+    /// [`set_control_flow()`]: EventLoopWindowTarget::set_control_flow()
+    /// [`run()`]: Self::run()
```

Comment:
Not super happy with this, because it'll fail to link on `not(all(wasm_platform, target_feature = "exception-handling"))`.

Reply:
`target_feature = "exception-handling"` isn't stable, which is probably why it didn't work for you.

Reply:
@daxpedda reading this again, I missed `target_` and assumed it was just `feature =` (from the `[features]` list in `Cargo.toml`).

I think the feature must have been _en_abled (running nightly rustdoc for `--cfg docsrs` => `doc_cfg`), as it's the only way to cause `not(all(true, true)) = not(true) = false` and turn _off_ `fn run()`?

Reply:
That is weird, `target_feature = "exception-handling"` should not automatically have been enabled by running nightly rustdoc.